web/package.json

```diff
@@ -65,6 +69,7 @@
     "lodash.get": "^4.4.0",
     "lodash.ismatch": "^4.4.0",
     "lodash.throttle": "^4.1.1",
+    "lucide-react": "^0.468.0",
```

Comment:
I believe we already have number of other icon sets and libs, why do we need one more?

Reply:
shadcn uses these internally to add icons to, e.g. dropdown component